src/protocols/rdp/rdp.h

```diff
@@ -48,6 +48,21 @@
 #include <pthread.h>
 #include <stdint.h>
 
+/**
+ * A flag for tracking if we are waiting conditionally on a username.
+ */
+#define GUAC_RDP_COND_FLAG_USERNAME 1
+
+/**
+ * A flag for tracking if we are waiting conditionally on a password.
+ */
+#define GUAC_RDP_COND_FLAG_PASSWORD 2
+
+/**
+ * A flag for tracking if we are waiting conditionally on a domain.
+ */
+#define GUAC_RDP_COND_FLAG_DOMAIN 3
```

Comment:
Since 3 is represented in binary as `11`, it cannot be safely used as the value of a bitwise flag intended to represent a state independent of the other flags. They would all need to be powers of two. I think you meant 4 here.

Reply:
Should be fixed.